scene/gui/text_edit.cpp

```diff
@@ -1831,6 +1831,8 @@ void TextEdit::gui_input(const Ref<InputEvent> &p_gui_input) {
 						queue_redraw();
 					}
 				} else if (drag_and_drop_selection_enabled && is_mouse_over_selection()) {
+					// In case the column length change during selection, make sure its up to date before assigning a new selection mode.
```

Comment:
```suggestion
					// In case the column length changes during selection, make sure it's up to date before assigning a new selection mode.
```

Reply:
Thank you for the correction dude. I am really bad at grammar. 🥲